test/functional/legacy/003_cindent_spec.lua

```diff
@@ -1832,6 +1852,26 @@ describe('cindent', function()
       {
       	111111111111111111;
       }
+      namespace test::cpp17
+      {
+      	111111111111111111;
+      }
+      namespace ::incorrectcpp17
+      {
+      	111111111111111111;
```

Comment:
Yes, I missed a tab and somehow managed not to notice that there are two identical places in the test file.
Thanks for a thorough review. Now the patches seem to be identical AFAICT. I noticed that there's a "lint" target so now I should be safe with that as well.